src/mscorlib/src/System/Collections/Generic/List.cs

```diff
@@ -976,8 +976,9 @@ public void Sort(IComparer<T> comparer)
             }
             Contract.EndContractBlock();
 
-            if( _size > 0) {
+            if (_size > 0) {
                 ArraySortHelper<T>.Sort(_items, 0, _size, comparison);
+                _version++;
```

Comment:
@ianhays Please see my new [commit,](https://github.com/dotnet/coreclr/pull/8944/commits/9a4f1619271a92958a1b4ef5e71e89e78fe50861) below in which I moved version increment out of the `if` block and added size checks in two `Sort` methods and also in the `Reverse` method. Is it ok now?